Disable code which passes ".machine ppc" to the assembler as it breaks
use of some CPU features. Notably breaks -maltivec which breaks build of
Mozilla ports including xulrunner.

Borrowed from https://github.com/sba1/adtools/commit/b24ae065714a2390d7718d2c348fe5cfa40c2b48

Index: gcc/config/rs6000/rs6000.c
--- gcc/config/rs6000/rs6000.c.orig
+++ gcc/config/rs6000/rs6000.c
@@ -6169,6 +6169,8 @@ rs6000_file_start (void)
   if (!(rs6000_default_cpu && rs6000_default_cpu[0])
       && !global_options_set.x_rs6000_cpu_index)
     {
+      /* Temporarily disabled as it overrides e.g., -mcpu=440 and -maltivec */
+#if 0
       fputs ("\t.machine ", asm_out_file);
       if ((rs6000_isa_flags & OPTION_MASK_MODULO) != 0)
 	fputs ("power9\n", asm_out_file);
@@ -6186,6 +6188,7 @@ rs6000_file_start (void)
 	fputs ("ppc64\n", asm_out_file);
       else
 	fputs ("ppc\n", asm_out_file);
+#endif
     }
 #endif
 
